sentry-sidekiq/spec/spec_helper.rb

```diff
@@ -1,6 +1,10 @@
 require "bundler/setup"
 require "pry"
 
+# This allows us to call sidekiq private methods for testing.
+# Make sure to set this before requiring sidekiq.
+$TESTING = true
```

Comment:
This global variable is used in the sidekiq codebase and allows us to call private methods. Using this can potentially improve the current code that uses `send` to call private methods.

Reply:
I think calling `Processor#process` with `send` in a few occasions is acceptable. so I prefer not to add this now.

thanks for sharing this information though, I never knew we can do that 🙂

Reply:
Do you think it's better to delete this to be consistent?

Reply:
yeah that'd be better, thanks 👍